config.yml

```diff
@@ -20,6 +20,20 @@ include: examples/autoload.php  # A file to include on each command
 #     rw_timeout: 60      # Redis read/write timeout
 #     phpiredis: false    # Should use phpiredis php extension (must be installed)
 
+# # the predis native way (sentinel example)
+# # "config" is the first parameter to Predis\Client
+# # "options" is the second parameter to Predis\Client
+# predis:
+#   config:
+#     - "tcp://10.0.0.1:12345"
+#     - "tcp://10.0.0.2:12345"
+#     - "tcp://10.0.0.3:12345"
+#   options:
+#     service: "some_redis_cluster_name"
+#     replication: "sentinel"
```

Comment:
That's true, so I'd just link to the [Predis documentation](https://github.com/nrk/predis/wiki/Connection-Parameters) instead of duplicating it on our own.

If someone wants to use advanced features they can do so, but the defaults in the config template should IMHO match the ones in the legacy configuration to avoid confusion.